apps/testing-javascript/src/trpc/routers/_app.ts

```diff
@@ -0,0 +1,15 @@
+import {
+  mergeRouters,
+  router,
+  skillLessonRouter,
+} from '@skillrecordings/skill-lesson'
+import {abilities} from './abilities'
+
+export const appRouter = mergeRouters(
+  router({
+    abilities: abilities,
+  }),
+  skillLessonRouter,
+)
+
+export type AppRouter = typeof appRouter
```

Comment:
@joelhooks I'm getting some unexpected type errors for the `appRouter`. I took after the implementation in total-typescript for this trimmed down app router implementation. Plus it was working in the `skill-template` apps I was generating. Any chance this looks familiar to you and/or you can help me understand how to fix it?

<img width="1011" alt="CleanShot 2023-01-25 at 15 45 00@2x" src="https://user-images.githubusercontent.com/694063/214698393-e7f2fc20-6ae4-4316-8416-da817f6197a2.png">

These are the errors I get when I run `pnpm lint`:

<img width="1490" alt="CleanShot 2023-01-25 at 15 41 50@2x" src="https://user-images.githubusercontent.com/694063/214698121-3dbb6dee-aa2f-4061-b47e-661b6b8ea81a.png">